app/src/ticketForm.js

```diff
@@ -0,0 +1,184 @@
+    this.client.addAlgoliaAgent(`Zendesk Integration (${version})`);
+    this.indexName = `${indexPrefix}${subdomain}_articles`;
+
+    if (clickAnalytics) {
+      initInsights({ applicationId, apiKey });
+      extendWithConversionTracking(this, {
+        clickAnalytics,
+        indexName: this.indexName,
+      });
+    }
+  }
+
+  init({
+    baseUrl,
+    clickAnalytics,
+    locale,
+    translations,
+    ticketForms: {
+      enabled,
+      inputSelector,
+      suggestionsListSelector,
+      descriptionSelector,
+      answersParameters,
+      requireSubject,
+      cssClasses: {
+        descriptionGroup,
+        disabledDescriptionGroup,
+        descriptionWarning,
+        suggestionsList,
+      },
+    },
+  }) {
+    if (!enabled) return;
+
+    // eslint-disable-next-line consistent-this
+    const self = this;
+
+    if (requireSubject) {
+      self.descriptionElement = document.querySelector(descriptionSelector);
+      self.descriptionElement.classList.add(descriptionGroup);
+      self.descriptionElement.classList.add(disabledDescriptionGroup);
+      const warning = document.createElement('span');
+      warning.classList.add(descriptionWarning);
+      warning.textContent = translate(translations, locale, 'descriptionLock');
+      self.descriptionElement.append(warning);
+    }
+
+    document
+      .querySelector(suggestionsListSelector)
+      .setAttribute('style', 'display: none!important');
+
+    const lang = locale.split('-')[0];
+    const buildUrl = (hit) => `${baseUrl}${locale}/articles/${hit.id}`;
+
+    const Input = () => {
+      const [subject, setSubject] = useState('');
+      const [answers, setAnswers] = useState([]);
+
+      const handleSubject = (e) => {
+        setSubject(e.target.value);
+        debounceGetAnswers({
+          index: self.client.initIndex(self.indexName),
+          query: e.target.value,
+          lang,
+          searchParams: {
+            facetFilters: `["locale.locale:${locale}"]`,
+            clickAnalytics,
+          },
+          answerParams: answersParameters,
+          callback: ({ hits, queryID }) => {
+            setAnswers(
+              hits.map((hit, i) => {
+                if (hit._answer.extractAttribute === 'body_safe') {
+                  hit._snippetResult.body_safe.value = hit._answer.extract;
+                }
+                hit.__position = i + 1;
+                hit.__queryID = queryID;
+                hit.url = buildUrl(hit);
+                return hit;
+              })
+            );
+            if (requireSubject) {
+              self.descriptionElement.classList.remove(
+                disabledDescriptionGroup
+              );
+            }
+          },
+          autocomplete: false,
+        });
+      };
+
+      const onClick = (e, item) => {
+        e.preventDefault();
+        self.trackClick(item, item.__position, item.__queryID);
+        window.location = e.target.href;
```

Comment:
I think you can just do 

```js
const onClick = (e, item) => { self.trackClick(item, item.__position, item.__queryID); }
```

and not prevent the default behavior which you seem to anyway reimplement with `
window.location = e.target.href;` isn't it?